package.json

```diff
@@ -1,14 +1,13 @@
 {
   "name": "fury-adapter-apiary-blueprint-parser",
-  "version": "1.1.3",
+  "version": "1.2.0",
```

Comment:
This should be a major version bump as it is introducing a backwards incompatible change (dropping Node versions)?